### PR TITLE
fixes #3: Created tomato soup file with ingredients and directions

### LINKED ISSUE #3: Tomato soup
I would like to add a simple tomato soup recipe!

### PR DESCRIPTION
**Pull request by j-bibbe**

### CODE REVIEW

**Comment by j-bibbe** [comment]
Does anyone know the English word for the little pieces of garlic? If so, please change it in the tomato-soup.md file ;)

File: soups/tomato-soup.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Tomato soup recipe
+
+## Ingredients
+- one onion, chopped
+- two pieces of garlic, chopped
```
Comment (bast) [inline soups/tomato-soup.md:5]:
```suggestion
- two garlic cloves, chopped

```

Comment (bast) [reply soups/tomato-soup.md:5]: I think this is the term ...

**Comment by j-bibbe** [comment]
Thanks @bast :)

**Pull request merged**